dev_tools/conf/mypy.ini

```diff
@@ -17,7 +17,8 @@ follow_imports_for_stubs = true
 
 #Adding "sympy.* or mypy-sympy to the above list (3rd-party libs for which we don't have stubs) doesn't ignore "cannot find module 'sympy' error
 [mypy-sympy.*]
-ignore_missing_imports = True
+follow_imports = skip
```

Comment:
This seems like it makes the type checking less robust for all the packages that do have good types.  It feels like we are losing a lot for one packages problems?

Reply:
This only skips following imports to the `sympy` package, which I think is basically equivalent to the status quo since sympy <= 1.9 does not ship with type annotations. If we change this to `follow_imports=silent` we end up with 475 type errors; it'd be good to start whittling those down, but not clear to me that this needs to happen before 1.0.

Reply:
ah I misread what this does, thanks